### DISCUSSION TITLE
How do I add goguardian back on my Chromebook

### DISCUSSION THREAD

**Discussion by Otaku012** [3kh0/ext-remover#101] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I want to add back goguardian but since it doesn't have a chrome web store page I can't. If anyone has a answer please let me know ASAP

**Comment by lurklite** [comment, marked as answer]
First of all. try using the alternates listed in other discussions. I dont remember them. 

or just restart your computer, but that means you won't be able to use it again. but that's probably your best option of the first option abovd doesn't work. just hope a new exploit for extentions gets released. cause I need it too
